### ISSUE TITLE
[a11y] Picklist options should be wider than the field

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1411]
![image](https://user-images.githubusercontent.com/37256050/167187767-a849dd38-27cb-4273-9116-61070a06162c.png)

![image](https://user-images.githubusercontent.com/37256050/167187808-18c48329-56b9-4002-9c5f-713bb6964074.png)
